docs/application/dotnet/guides/nui/texteditor.md

```diff
@@ -0,0 +1,113 @@
+| `CursorWidth`                      | Integer     | The cursor width                         |
+| `GrabHandleImage`                  | String      | The image to display for the grab handle |
+| `GrabHandlePressedImage`           | String      | The image to display when the grab handle is pressed |
+| `SelectionHandleImageLeft`         | PropertyMap | The image to display for the left selection handle |
+| `SelectionHandleImageRight`        | PropertyMap | The image to display for the right selection handle |
+| `SelectionHandlePressedImageLeft`  | PropertyMap | The image to display for the left selection handle marker |
+| `SelectionHandlePressedImageRight` | PropertyMap | The image to display for the right selection handle marker |
+| `SelectionHandleMarkerImageLeft`   | PropertyMap | The image to display for the left selection handle marker |
+| `SelectionHandleMarkerImageRight`  | PropertyMap | The image to display for the right selection handle marker |
+| `SelectionHighlightColor`          | Vector4     | The color of the selection highlight     |
+| `DecorationBoundingBox`            | Rectangle   | The decorations (handles etc) will positioned within this area on-screen |
+| `EnableMarkup`                     | Boolean     | Whether the mark-up processing is enabled |
+| `InputColor`                       | Vector4     | The color of the new input text          |
+| `InputFontFamily`                  | String      | The font's family of the new input text  |
+| `InputFontStyle`                   | PropertyMap | The font's style of the new input text   |
+| `InputPointSize`                   | Float       | The font's size of the new input text in points |
+| `LineSpacing`                      | Float       | The default extra space between lines in points |
+| `InputLineSpacing`                 | Float       | The extra space between lines in points  |
+| `Underline`                        | PropertyMap | The default underline parameters         |
+| `InputUnderline`                   | String      | The underline parameters of the new input text |
+| `Shadow`                           | PropertyMap | The default shadow parameters            |
+| `InputShadow`                      | String      | The shadow parameters of the new input text |
+| `Emboss`                           | String      | The default emboss parameters            |
+| `InputEmboss`                      | String      | The emboss parameters of the new input text |
+| `Outline`                          | PropertyMap | The default outline parameters           |
+| `InputOutline`                     | String      | The outline parameters of the new input text |
+| `SmoothScroll`                     | Boolean     | Enable or disable the smooth scroll animation |
+| `SmoothScrollDuration`             | Float       | Sets the duration of smooth scroll animation |
+| `EnableScrollBar`                  | Boolean     | Enable or disable the scroll bar         |
```

Comment:
Suggestion : We are on text editor page now (and the table is about texteditor properties), so I think it would be ok not to emphasize `on text editor` again. What do you think?

Reply:
@mijin85cho, I axm okay with your suggestion.
 "text editor can be removed from the sentence"